phpcs.xml.dist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset>
+    <arg name="basepath" value="."/>
+    <arg name="extensions" value="php"/>
+    <arg name="parallel" value="45"/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors"/>
+
+     <!-- Ignore warnings and show progress of the run -->
+    <arg value="np"/>
+
+    <file>Command</file>
```

Comment:
Instead of explicitly listing files, we should include `.` and exclude the `vendor` folder. Otherwise we won't check the files that are in the root folder of the bundle.